gui/default/syncthing/settings/advancedSettingsModalView.html

```diff
@@ -1,77 +1,77 @@
 <modal id="advanced" status="danger" icon="fas fa-cogs" heading="{{'Advanced Configuration' | translate}}" large="yes" closeable="yes">
   <div class="modal-body">
 
-    <p class="text-danger">
-      <b translate>Be careful!</b>
-      <span translate>Incorrect configuration may damage your folder contents and render Syncthing inoperable.</span>
-    </p>
+      <p class="text-danger">
+        <b translate>Be careful!</b>
+        <span translate>Incorrect configuration may damage your folder contents and render Syncthing inoperable.</span>
+      </p>
```

Comment:
Too much indent, was fine before.
```suggestion
    <p class="text-danger">
      <b translate>Be careful!</b>
      <span translate>Incorrect configuration may damage your folder contents and render Syncthing inoperable.</span>
    </p>
```

Reply:
Done.